modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientExperimentalCodegen.java

```diff
@@ -375,41 +357,6 @@ public void updateCodegenPropertyEnum(CodegenProperty var) {
         // overwriting defaultValue omitted from here
     }
 
-    @Override
-    public CodegenParameter fromRequestBody(RequestBody body, Set<String> imports, String bodyParameterName) {
-        CodegenParameter result = super.fromRequestBody(body, imports, bodyParameterName);
-        // if we generated a model with a non-object type because it has validations or enums,
-        // make sure that the datatype of that body parameter refers to our model class
-        Content content = body.getContent();
-        Set<String> keySet = content.keySet();
-        Object[] keyArray = (Object[]) keySet.toArray();
-        MediaType mediaType = content.get(keyArray[0]);
-        Schema schema = mediaType.getSchema();
-        String ref = schema.get$ref();
-        if (ref == null) {
-            return result;
-        }
-        String modelName = ModelUtils.getSimpleRef(ref);
-        // the result lacks validation info so we need to make a CodegenProperty from the schema to check
-        // if we have validation and enum info exists
-        Schema realSchema = ModelUtils.getSchema(this.openAPI, modelName);
-        CodegenProperty modelProp = fromProperty("body", realSchema);
-        if (modelProp.isPrimitiveType && (modelProp.hasValidation || modelProp.isEnum)) {
-            String simpleDataType = result.dataType;
-            result.isPrimitiveType = false;
-            result.isModel = true;
-            result.dataType = modelName;
-            imports.add(modelName);
-            // set the example value
-            if (modelProp.isEnum) {
-                String value = modelProp._enum.get(0).toString();
-                result.example = modelName + "(" + toEnumValue(value, simpleDataType) + ")";
-            } else {
-                result.example = modelName + "(" + result.example + ")";
-            }
-        }
-        return result;
-    }
```

Comment:
We can delete this because it was only responsible for setting the result.example
That is now handled by setParameterExampleValue